perftracker/static/js/perftracker.js

```diff
@@ -990,4 +990,81 @@ function pt_isInViewport(elem) {
 
 function pt_pluralize(count, noun, suffix = 's') {
     return `${count} ${noun}${count !== 1 ? suffix : ''}`;
+}
+
+function pt_charts_analyze() {
+    if (hide_charts_information) {
+        alert('Not implemented yet!');
+        return;
+    }
+    if (analyze_was_called) return;
+    analyze_was_called = true;
+
+    var payload = {};
+    Object.keys(sect_data).forEach(s_id => {
+        payload[s_id] = sect_data[s_id].series['0'].data;
```

Comment:
While I usually recommend to move static code to static file, this is an exception.
The code depends on 'sect_data' variable, and some other variables _not_ defined in perftracker.js, but actually generated by specific view.
This looks strange and misleading and hard to investigate. Either pass that variables as arguments, or embed this code to specific view.